src/components/Input/Input.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styles from './Input.module.scss';
+import PropTypes from 'prop-types';
+
+const Input = ({ tag: Tag, name, type, label, maxLength, ...props }) => (
+	<div className={styles.inputWrapper}>
+		<input
+			className={Tag !== 'searchInput' ? styles.input : styles.searchInput}
+			type={type}
+			name={name}
+			id={name}
+			placeholder=" "
+			maxLength={maxLength}
+			{...props}
+		/>
+		<label className={styles.label} htmlFor={name}>
+			{label}
+		</label>
+		<div className={styles.inputBar}></div>
+	</div>
+);
+
+Input.propTypes = {
+	tag: PropTypes.string,
```

Comment:
If `tag` can take multiple values from a limited set, list those possible values with `oneOf` instead of just `string`